File: QuickParked-Back/src/main/java/com/example/quickparkedback/Model/Typevehicle.java
package com.example.quickparkedback.Model;

import com.example.quickparkedback.Model.Enums.TypeCar;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import static javax.persistence.GenerationType.AUTO;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Typevehicle {
    @Id
    private Integer id;
    private String typevehicle;
    private Integer rate;
}

File: QuickParked-Front/src/app/models/TypeVehicle.ts
export class TypeVehicle{
    id?: number;
    typevehicle: string;
    rate: number;

    constructor(_id: number, _typevehicle: string, _rate: number){
        this.id = _id;
        this.rate = _rate;
        this.typevehicle = _typevehicle;
    }
}